daemon/src/engine/client/client.h

```diff
@@ -74,11 +74,10 @@ typedef struct
 	int           cmdNum; // the next cmdNum the server is expecting
 	playerState_t ps; // complete information about the current player at this time
 
-	unsigned      numEntities; // all of the entities that need to be presented
-	int           parseEntitiesNum; // at the time of this snapshot
-
 	int           serverCommandNum; // execute all commands up to this before
 	// making the snapshot current
+
+    std::vector<entityState_t> entities;
```

Comment:
:( does it mean that inside a single code block indentation has to be consistent?